app/models/user.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class User < ActiveRecord::Base
+  has_many :reviews
```

Comment:
We should search if there's a way to specify if the user is the reviewer or the reviewee.